src/script/externalRoute.ts

```diff
@@ -62,6 +64,13 @@ export const getWebsiteUrl = (path: string = '', pkCampaign?: string): string |
   }
   return undefined;
 };
+export const getHelpCenterUrl = (path: string = ''): string | undefined => {
```

Comment:
Would be nice to be more strict about types here, eg. add `as const` to `URL_PATH` and use type `typeof URL_PATH[keyof typeof URL_PATH]`

Reply:
good idea, I need to clean up the rest of the file anyway, I think there's a whole bunch of unused links